client/containers/DashboardSettings/CommunitySettings.tsx

```diff
@@ -342,6 +342,24 @@ const CommunitySettings = () => {
 						/>
 					</InputField>
 				</div>
+				<div className="row-wrapper">
+					<a href="/dash/scripts">Customize CSS</a>
+					<div style={{ paddingLeft: '5px' }}>
```

Comment:
Let's avoid using inline CSS here, and use the class name props that `Tooltip` provides instead:

```tsx
<Tooltip
    targetClassName="custom-css-tooltip-target"
    popoverClassName="dashboard-settings-container_custom-css-popover"
    content={
        <>
            Warning: Experimental...
        </>
    }
>
```

Then you can add a `.custom-css-tooltip-target` selector inside of `.dashboard-settings-container` to add that 5px of left margin. You'll then need to add a _top-level_ selector `.dashboard-settings-container_custom-css-popover` to hold the 400px max-width property. This gets rendered in a React portal, so it's annoyingly not a child of `.dashboard-settings-container`. Its long class name with the `_` separating the two parts is a loose convention of ours for naming these portalled children-but-not-really.

In general when adding styles that are statically known, we should hew towards using SCSS rather than inline styles. I think we can bend this rule when it's exceedingly inconvenient; this is an example of where it's kinda inconvenient, but there are patterns in place to make it work :)